docs/core/howto/listings/systemd/www.example.com.socketactivated.service

```
@@ -5,7 +5,7 @@ Description=Example Web Server
 ExecStart=/usr/bin/twistd \
     --nodaemon \
     --pidfile= \
-    web --listen systemd:domain=INET:index=0 --path .
+    web --listen systemd:domain=INET:name=www --path .
```

It looks like the prose and socket unit call this `my-web-port`:

```suggestion
    web --listen systemd:domain=INET:name=my-web-port --path .
```